packages/react/runtime/src/lynx.ts

```diff
@@ -41,7 +41,7 @@ if (__DEV__) {
 }
 
 // TODO: replace this with __PROFILE__
-if (__PROFILE__) {
+if (__PROFILE__ || lynx.performance?.isProfileRecording?.()) {
```

Comment:
`isProfileRecording` is a runtime API, so will `initProfileHook` always be bundled after this change?

Reply:
Yes. As the title says, we need to enable profile in production build